hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -263,10 +263,10 @@
 DEFAULT_FROM_EMAIL = "webmaster@localhost"
```

Comment:
Do we need to change this?

Reply:
changed in another PR for IEEE-154 (just got merged)